### DISCUSSION TITLE
Behavior(not displayed) of text labels on treemap

### DISCUSSION THREAD

**Discussion by mastavichus-vp** [kurkle/chartjs-chart-treemap#62] [Q&A] [answered]
Please help understand the behavior of text labels. 
In examples on
[https://chartjs-chart-treemap.pages.dev/samples/labels.html](https://chartjs-chart-treemap.pages.dev/samples/labels.html)
all OK, labels displayed.
But this:
[https://codepen.io/mastavichus-vp/pen/MWvZzMw](https://codepen.io/mastavichus-vp/pen/MWvZzMw)
labels not displayed, strangely ((

**Comment by kurkle** [comment, marked as answer]
The docs are not versioned (yet), so they reflect current development in master. Labels are not yet released to NPM.

There are breaking changes, so next release will be 2.0.0.
https://github.com/kurkle/chartjs-chart-treemap/milestone/2
